### PR TITLE
Source Hubspot - fix missing data issue

### PR DESCRIPTION
**Pull request by davydov-d**
## What
https://github.com/airbytehq/oncall/issues/278

## How
After the fix https://github.com/airbytehq/airbyte/pull/13596 was introduced, we started passing list of properties as a `property` parameter unconditionally. This should be done depending on the API version: `property` for v1, `properties` - for v3

### CODE REVIEW

**Review by grubberr — APPROVED** [review]

**Review by arsenlosenko — APPROVED** [review]

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/helpers.py [hunk 1]
```diff
@@ -51,3 +52,62 @@ def add_record(self, record: MutableMapping):
...
+
+    def as_url_param(self):
+        return {"property": self.properties}
+
+
+class APIv3Property(IURLPropertyRepresentation):
+    _term_representation = "{property},"
+
+    def as_url_param(self):
+        return {"properties": ",".join(self.properties)}
```
Comment (evantahler) [inline airbyte-integrations/connectors/source-hubspot/source_hubspot/helpers.py:113]: Awesome!

**Review by evantahler — APPROVED** [review]

**Comment by evantahler** [comment]
I will publish and merge

**Comment by evantahler** [comment]
Why test so many other connectors in this PR?

**Comment by davydov-d** [comment]
> Why test so many other connectors in this PR?

I'm sorry, I just needed a place to rerun a couple of builds. Ignore them please

**Pull request merged**